opensuse/README-short.txt

```diff
@@ -1 +1 @@
-This project contains the stable releases of the openSUSE distribution.
+DEPRECATED - for current official images see opensuse/leap and opensuse/tumbleweed.
```

Comment:
I think using the term "official images" here might cause some confusion; how about this instead?

```suggestion
DEPRECATED - for current openSUSE images see opensuse/leap and opensuse/tumbleweed.
```

Reply:
The main point of this is to tell users that the official images made by the openSUSE project can be found elsewhere - so I'm against removing "official images" in the short description.

Reply:
I reworded it, should be fine now